extensions/amp-form/0.1/amp-form.js

```diff
@@ -99,6 +150,11 @@ export class AmpForm {
   /** @private */
   installSubmitHandler_() {
     this.form_.addEventListener('submit', e => this.handleSubmit_(e), true);
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
So, one "small" problem is that if we decide to add any custom elements that will create `<input type=hidden>` or other types of input in their `buildCallback`, this method maybe executed too early to pick them all up. Possibly enough to file a bug for now, but something to fix down the road.